hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/strategies/VersionedEngagementsStrategy.java

```diff
@@ -101,14 +101,20 @@ public List<Finding> run(Scan scan, ScanFile scanResultFile) throws Exception {
 
     ScanType scanType = ScanNameMapping.bySecureCodeBoxScanType(scan.getSpec().getScanType()).scanType;
     TestType testType = testTypeService.searchUnique(TestType.builder().name(scanType.getTestType()).build()).orElseThrow(() -> new DefectDojoPersistenceException("Could not find test type '" + scanType.getTestType() + "' in DefectDojo API. DefectDojo might be running in an unsupported version."));
-    
+
+    var additionalValues = new LinkedMultiValueMap<String, Object>();
```

Comment:
The function reimportScan, which is part of the [defectdojo-client-java](https://github.com/secureCodeBox/defectdojo-client-java/tree/main), requires a LinkedMultiValueMap<String, Object>. We could change the type to LinkedMultiValueMap<String, String> since, as far as i can see, only strings are used for the object type. This would require further changes in the defectdojo-client-java.

Reply:
If I change it testwise to type to LinkedMultiValueMap<String, String> and try to build it (gradlew build), I get and exception for incompatible types.

Reply:
Let's fix this in a new PR.